sympy/integrals/integrals.py

```diff
@@ -41,6 +43,8 @@ class Integral(AddWithLimits):
 
     __slots__ = ('is_commutative',)
 
+    args: tTuple[Expr, Any]
```

Comment:
Why is Any here?

Reply:
https://github.com/sympy/sympy/blob/6a5cb5dabd6d090cc1d45f7ef591462cb0026fc6/sympy/core/evalf.py#L1018-L1020
This expects the second arg to be a trinary `sympy.core.containers.Tuple`, which is guaranteed **dynamically** at
https://github.com/sympy/sympy/blob/6a5cb5dabd6d090cc1d45f7ef591462cb0026fc6/sympy/core/evalf.py#L1114-L1117
but no type hints can be **statically** applied.

Reply:
Can it not be `tTuple[Expr, Tuple[Expr, ...]]`?

I guess mypy doesn't understand that Tuple is like tuple so maybe `tTuple[Expr, Tuple]` with the implicit understanding that the args of Tuple are Basic.

Reply:
`tTuple[Expr, Tuple]` is feasible. I didn't notice `Integral(x, x)` automatically converts the second arg to sympy tuple.
`Tuple[Expr, ...]` produces
```
sympy/integrals/integrals.py:46: error: Unexpected "..."
sympy/integrals/integrals.py:46: error: "Tuple" expects no type arguments, but 2 given
```